lib/hammer_cli_katello/host_collection.rb

```diff
@@ -51,15 +51,47 @@ class InfoCommand < HammerCLIKatello::InfoCommand
     end
 
     class HostsCommand < HammerCLIKatello::ListCommand
-      resource :host_collections, :hosts
+      resource :hosts, :index
       command_name "hosts"
 
+      option "--id", "HOST_COLLECTION_ID", _("Host Collection ID")
+      option "--name", "HOST_COLLECTION_NAME", _("Host Collection Name")
+
+      validate_options do
+        host_collection_options = [:option_id, :option_name]
+        organization_options = [:option_organization_id, :option_organization_name,
+                                :option_organization_label]
+
+        any(*host_collection_options).required
+
+        if option(:option_name).exist?
```

Comment:
For some reason if I do that then the exception ends up being `EX_SOFTWARE` instead of `EX_USAGE` in the test.